lib/src/camera.dart

```diff
@@ -0,0 +1,45 @@
+import "package:burt_network/burt_network.dart";
+import "package:opencv_ffi/opencv_ffi.dart";
+import "dart:async";
+import "collection.dart";
+
+class CameraManager {
+  late final Camera camera;
+  late final CameraDetails details;
+  Timer? timer;
+  
+  CameraManager({required this.camera, required this.details}){ 
+    init();
+  }
```

Comment:
```suggestion
```
You'd want to keep the `init` function separate so it can be initialized whenever the creator wants to. Otherwise you might as well do all the work in the constructor. (Also, if `init` is `async`, then it can be `await`ed)

Reply:
pass in camera and details?

Reply:
Yep. You can even pass in just the details and the manager can use the name to create the camera